scripts/dumpapp

```diff
@@ -26,8 +26,14 @@ def main():
   # transport.
   device = os.environ.get('ANDROID_SERIAL')
 
+  # Connect on the overridden port
+  portStr = os.environ.get('ANDROID_ADB_SERVER_PORT', '')
+  port = 5037
+  if portStr.isdigit():
```

Comment:
If `ANDROID_ADB_SERVER_PORT` is not an integer this should throw. adb itself does something similar.